codalab/scripts/initialize.py

```diff
@@ -168,6 +169,24 @@ def insert_data():
     for name, codename in submission_status_set:
         _, _ = CompetitionSubmissionStatus.objects.get_or_create(name=name, codename=codename)
 
+    team_status = [("Denied", "denied", "Team was denied."),
```

Comment:
Is just a code to run when no teams apps where installed. Is just the definition of the constants, as was done before in Codalab. Maybe now is done in migrations, therefore it should be the same case, I guess. If you want I check something, please tell me